scripts/report-test-results.py

```diff
@@ -0,0 +1,112 @@
+import os
+import sys
+from xml.dom.minidom import Node, parse
+
+
+def main():
+    if len(sys.argv) < 3:
+        print("Usage: python report-test-results.py working-dir testsuite-list")
+        sys.exit(0)
+
+    working_dir = sys.argv[1]
+    test_file_names = sys.argv[2:]
+    test_files = [
+        working_dir + "/functional-test-reports/" + filename
+        for filename in test_file_names
+    ]
+
+    if not len(test_files):
+        print("Please provide a list of test files")
+        sys.exit(0)
+
+    artefact_bucket = get_smoke_test_bucket_name()
+
+    for test_file in test_files:
+        test_results = extract_test_result(parse(test_file))
+        for r in test_results:
+            success = "PASS" if len(r) < 4 else "FAIL"
+
+            result = f"PRV-FT-{r[0].upper()} | {success} | NAME: {r[1]} | TIME: {r[2]}"
+
+            if len(r) > 3:
+                result = result + f" | ERROR: {r[3]} | FILE: {r[4]}"
+                if artefact_bucket is not None:
+                    result = (
+                        result
+                        + " | Output and screenshots saved to S3 bucket "
+                        + f"https://eu-west-2.console.aws.amazon.com/s3/buckets/{artefact_bucket}"
```

Comment:
Just something for consideration here. I probably wouldn't publish a full link in this way, I would maybe state the account and the bucket name, and leave the user to make their way there. Only reason for that is that if I was, for instance, logged into the production console and followed this link, it would look for the bucket in prod and it wouldn't be there. This is an annoying thing about these links, they don't specify the actual AWS account.